python/paddle/fluid/dygraph/dygraph_to_static/ifelse_transformer.py

```diff
@@ -656,46 +685,43 @@ def transform_if_else(node, root):
     return true_func_node, false_func_node, return_name_ids
 
 
-def create_cond_node(return_name_ids, pred, true_func, false_func):
+def create_cond_node(return_name_ids,
+                     pred,
+                     true_func,
+                     false_func,
+                     is_if_expr=False):
     """
     Create `fluid.layers.cond(pred, true_fn, false_fn)` to replace
     original `python if/else` statement.
     """
+
+    def create_lambda_node(func_node, is_if_expr=False):
```

Comment:
NIT: func_node is not a function name when `is_if_expr=True`. So I suggest to rename `func_node` to `func_or_expr_node`

Reply:
Thanks, fixed.